src/Data/List/Extra.hs

```diff
@@ -455,6 +457,26 @@ groupSortBy :: (a -> a -> Ordering) -> [a] -> [[a]]
 groupSortBy f = groupBy (\a b -> f a b == EQ) . sortBy f
 
 
+-- | A strict version of 'sum'.
+sum' :: (Num a) => [a] -> a
```

Comment:
Could we add the example `sum' [1, 2, 3] == 6` just so people have a concrete example to go from.

Reply:
Done!